### PR TITLE
add enterprise and reorg permissions

### PR DESCRIPTION
**Pull request by nataliejeremy**
@bmackinney @slivermon @ari-gold Please review--thanks!

### CODE REVIEW

File: source/docs/articles/organizations/change-management.md [hunk 1]
```diff
@@ -54,74 +55,127 @@ Change Management is enabled for all Pantheon for Agencies Partners, and for som
...
+        <td>Administrator</td>
+        <td>- Manage settings<br>
+        - Remove members    <br>
+        - Edit member roles <br>
+        - Delete sites  <br>
+        - Remove sites from the Organization</td>
+    </tr>
+    <tr>
+        <td class="border-bottom">Team Member</td>
+        <td class="border-bottom">- Deploy code<br/>- Assign a "User in Charge": This is the person who created a site and is able to delete it.</td>
```
Comment (bmackinney) [inline source/docs/articles/organizations/change-management.md]: This is only called "User in Charge" in enterprise One orgs, I believe? @bensheldon @slivermon may be able to clarify.

**Comment by bmackinney** [comment]
Sorry so slow!!! This looks great. Only the one question. then :+1:

**Comment by bensheldon** [comment]
> This is only called "User in Charge" in enterprise One orgs

Correct. 

Agencies will see Owner column.

**Comment by bensheldon** [comment]
Also, you cannot change the User in Charge. It's the user who created the site, unless they have been removed from the team, in which case it is blank

**Comment by nataliejeremy** [comment]
Thanks @bensheldon I removed "user in charge" from the Partner Org table and Supporting Org table.

**Comment by rachelwhitton** [comment]
+1

**Comment by bmackinney** [comment]
@nataliejeremy I modified some of the language and I think we can possibly merge the tables for site-level permissions. The headings also now don't look great but are more accurate. Do you mind if I bounce this PR back to you?

**Comment by nataliejeremy** [comment]
@bmackinney Sure, but I like the idea of keeping the tables separate though, so it's clear for each org

**Comment by bmackinney** [comment]
I failed to explain why. The permissions are the same at the site level, for Admins, Team Members, and Developers. The only difference is the possibility of "Owner" vs "User in Charge"

**Comment by bmackinney** [comment]
Needs more work to account for changing permissions after site is launched, restricting admin ability to delete.

**Comment by nataliejeremy** [comment]
@bmackinney is that the only permission that changes after the site is launched?

**Comment by nataliejeremy** [comment]
@bmackinney if an admin can't delete a site after launch, which role can?

**Comment by bmackinney** [comment]
@nataliejeremy I need to clarify: this *should* only apply to partner org admins, and only the site owner should be able to delete the site after it goes live.

**Comment by bmackinney** [comment]
@nataliejeremy can you follow up on the last few comments with @slivermon and/or CSE and fix the conflict?

File: source/docs/articles/organizations/change-management.md [hunk 2]
```diff
@@ -6,46 +6,59 @@ category:
...
 
 ## Features
-* Organizations with Change Management can add people as Administrators, Team Members, and Developers to the Organization. Admins have all privileges; Team Members are a trusted role that can make changes that affect the Test and Live environments; and Developers can only work on Development environments.
-* When an Organization is added to a site as a supporting Organization (e.g. Agencies), the members and roles of the Organization are inherited for working on the site. All Organization members can access all sites associated with the organization.
-* Agencies added to Sites by subscription owners are able to use Change Management internally, but the subscription owners are not—they can only add full team members.
+* Enterprise organization administrators and site team members can add Users or Supporting Organizations to sites owned by Organization with the Developer or Team Member roles.
```
Comment (bensheldon) [inline source/docs/articles/organizations/change-management.md]: I think this maybe should clarify that "Users" means "Team Members". e.g.

> Enterprise organization administrators and site team members can add Team Members or Supporting Organizations to sites owned by Organization with the Developer or Team Member roles.

File: source/docs/articles/organizations/change-management.md [hunk 3]
```diff
@@ -6,46 +6,59 @@ category:
...
-* When an Organization is added to a site as a supporting Organization (e.g. Agencies), the members and roles of the Organization are inherited for working on the site. All Organization members can access all sites associated with the organization.
-* Agencies added to Sites by subscription owners are able to use Change Management internally, but the subscription owners are not—they can only add full team members.
+* Enterprise organization administrators and site team members can add Users or Supporting Organizations to sites owned by Organization with the Developer or Team Member roles. 
+* Partner Organizations can have users with a role: Administrator, Team Member, or Developer. 
+* When an Organization is added to a site as a Supporting Organization (e.g. Agencies), the members and roles of the Organization are inherited for working on the site, unless the site is owned by an Enterprise Organization with change management enabled. In this case, the site-level role is respected for all users in the Supporting Organization.
```
Comment (bensheldon) [inline source/docs/articles/organizations/change-management.md]: I would suggest rewording this:

> When an Organization is added to a site as a Supporting Organization (e.g. an Agency Organization), the members and roles of the Organization are inherited for working on the site. If the site is owned by an Enterprise Organization with Change Management enabled, the site administrators can "cap" the permission level of all Supporting Organization Users to "Developer", e.g. Supporting Organization Admins will only have the permissions level of a Developer when working on that site, despite being Organization Admins.

File: source/docs/articles/organizations/change-management.md [hunk 4]
```diff
@@ -6,46 +6,59 @@ category:
...
+* Agencies added to sites by subscription owners are able to use Change Management internally.
```
Comment (bensheldon) [inline source/docs/articles/organizations/change-management.md]: I think this line should be removed because it's not clear that "internally" means "On the organization Dashboard"

File: source/docs/articles/organizations/change-management.md [hunk 5]
```diff
@@ -6,46 +6,59 @@ category:
...
 * All users in an Organization can see other users and their roles.
 
+## Managing an Organization's People
```
Comment (bensheldon) [inline source/docs/articles/organizations/change-management.md]: Ugh, we should just rename the "People" tab to be "Users".

**Comment by nataliejeremy** [comment]
@bmackinney I'm also having Shawn review. I know you made a lot of changes...do you want me to make the updates/revisions from this point on?

File: source/docs/articles/organizations/change-management.md [hunk 6]
```diff
@@ -54,43 +67,63 @@ Change Management is enabled for all Pantheon for Agencies Partners, and for som
...
-</tr>
+    <tr>
+        <th><strong>Role</strong></th>
+        <th><strong>Permissions</strong></th>
+    </tr>
+    <tr>
+        <td>Administrator</td>
+        <td>* Manage settings<br>
+        * Remove members    <br>
+        * Edit member roles <br>
```
Comment (slivermon) [inline source/docs/articles/organizations/change-management.md]: Recommend using the same language as Partners in Enterprise for consistency.

"* Change member roles"

All roles can create sites.

File: source/docs/articles/organizations/change-management.md [hunk 7]
```diff
@@ -99,29 +132,43 @@ By default, Change Management is not enabled for some types of Organizations. If
...
+    </tr>
+    <tr>
+        <td>Team Member</td>
+        <td>* Deploy code to Test and Live<br/>* Use the Workflow tool on Test and Live<br/>* Change the PHP versions for all site environments<br/>* Add Team Members and Supporting Organizations</td>
+        <td><center>X</center></td>
+        <td><center>X</center></td>
+    </tr>
+    <tr>
+        <td>Developer</td>
+        <td>* Commit code to Dev and Multidev environments<br/>* Add and Remove add-ons to sites. New Relic cannot be removed once added.</td>
```
Comment (slivermon) [inline source/docs/articles/organizations/change-management.md]: Can we clarify a bit:

* Can only commit code to Dev and Multidev...

File: source/docs/articles/organizations/change-management.md [hunk 8]
```diff
@@ -99,29 +132,43 @@ By default, Change Management is not enabled for some types of Organizations. If
...
+        <td><center>X</center></td>
+        <td><center>X</center></td>
+    </tr>
+    <tr>
+        <td>User in Charge</td>
```
Comment (slivermon) [inline source/docs/articles/organizations/change-management.md]: Should note that User in Charge is not an assignable role; it's automatically assigned to whomever created the site until an owner takes over.

File: source/docs/articles/organizations/change-management.md [hunk 9]
```diff
@@ -99,29 +132,43 @@ By default, Change Management is not enabled for some types of Organizations. If
...
+    <tr>
+        <td>Owner</td>
+        <td>* Functionally is the same as an Organization Administrator</td>
+        <td></td>
+        <td><center>X</center></td>
+    </tr>
 </table>
+
+<br>
+To promote a Developer to the Team Member role for a single site, add the user to the Site Team.
```
Comment (slivermon) [inline source/docs/articles/organizations/change-management.md]: This is a great tip. May want to consider making it move obvious that it's a tip. 

Completely optional, but better still, talk through some use cases of how how CM works in an org.

**Comment by bmackinney** [comment]
All of my edits are here in this branch already

**Comment by slivermon** [comment]
Regarding who can delete a live (purchased) site, only the owner can delete it.

File: source/docs/articles/organizations/change-management.md (hunk 5 above)
Comment (nataliejeremy) [inline source/docs/articles/organizations/change-management.md]: Thanks @bensheldon I edited that.

**Comment by nataliejeremy** [comment]
@bensheldon @slivermon Thanks for reviewing. I incorporated the comments; please review one last time. Thanks!

File: source/docs/articles/organizations/change-management.md (hunk 8 above)
Comment (slivermon) [inline source/docs/articles/organizations/change-management.md]: After chatting with Brian, we recommend removing the 'User in Charge' role from the table because it's not user-assignable. 

Instead, it could be described outside the table (maybe below it) to simply describe that's it is just a label for showing who created the site which is helpful for enterprise orgs with unprivileged users like universities. 

Suggestion:
"In some dashboards, you may notice the 'User in Charge' label applied to a user. The 'User in Charge' is a label that helps distinguish who created a site in enterprise organizations with unprivileged users (e.g. Universities)."

**Comment by nataliejeremy** [comment]
@bmackinney all comments have been incorporated.

**Comment by slivermon** [comment]
Looks good. +1

File: source/docs/articles/organizations/change-management.md [hunk 10]
```diff
@@ -54,43 +68,63 @@ Change Management is enabled for all Pantheon for Agencies Partners, and for som
...
+        <td>Administrator</td>
+        <td>* Manage settings<br>
+        * Remove members    <br>
+        * Change member roles <br>
+        * Create and delete sites  <br>
+        * Remove sites from the Organization</td>
+    </tr>
+    <tr>
+        <td class="border-bottom">Unprivileged</td>
+        <td class="border-bottom">* Create sites, for which they automatically become the User in Charge until the site is assigned an owner.<br/></td>
```
Comment (bmackinney) [inline source/docs/articles/organizations/change-management.md]: They will always remain user in charge, and the org owns the site always.

File: source/docs/articles/organizations/change-management.md [hunk 11]
```diff
@@ -54,43 +68,63 @@ Change Management is enabled for all Pantheon for Agencies Partners, and for som
...
-</tr>
+
+    <tr>
+        <th><strong>Role</strong></th>
+        <th><strong>Permissions</strong></th>
+    </tr>
+    <tr>
+        <td>Administrator</td>
+        <td>* Manage settings<br>
+        * Remove members    <br>
```
Comment (bmackinney) [inline source/docs/articles/organizations/change-management.md]: I had a comment about using * for bullets. Can we end this and use `& #8226; or & bull;` - unicode bullets?
